modules/apps/asset/asset-list-web/src/main/resources/META-INF/resources/view_info_collection_providers.jsp

```diff
@@ -42,10 +42,16 @@ InfoCollectionProviderDisplayContext infoCollectionProviderDisplayContext = (Inf
 					<%= HtmlUtil.escape(infoCollectionProviderDisplayContext.getTitle(infoCollectionProvider)) %>
 
 					<c:if test="<%= infoCollectionProvider instanceof BetaInfoCollectionProvider %>">
-						<clay:label
-							displayType="info"
-							label='<%= LanguageUtil.get(request, "beta") %>'
-						/>
+						<span>
+							<react:component
```

Comment:
@ethib137 Could be possible to create a clay taglib for it?

Reply:
Having this kind of logic around portal is not easy to maintain and reuse.

Reply:
Hey @ealonso  Yeah, I totally agree and we will. Once we create this we will also migrate all these instances.

I've created this story to track it: https://liferay.atlassian.net/browse/LPS-204695

Reply:
Great!! 👏🏾